### PR TITLE
Add description for AllowTownhalls Policy in TeamsEvents Policy

### PR DESCRIPTION
**Pull request by navirdee**
Added description for AllowTownhalls policy

### CODE REVIEW

**Comment by navirdee** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by navirdee** [comment]
@microsoft-github-policy-service agree

**Review by get-itips — CHANGES_REQUESTED** [review]
please also add the parameter to the parameter syntax section

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@navirdee

**Comment by yogkumgit** [comment]
@navirdee Gentle reminder.

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -47,6 +47,26 @@ Possible values are:
...
+Aliases:
+
+Required: True
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowTownhalls
```
[the remark this answers is not in the document]
Comment (navirdee) [reply teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:62]: Fixed.

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -69,6 +69,56 @@ Accept wildcard characters: False
...
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowTownhalls
```
[the remark this answers is not in the document]
Comment (navirdee) [reply teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:84]: Done.

**Comment by yogkumgit** [comment]
@get-itips @tseward Please review. Thanks

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**